examples/consumer/manual/main.go

```diff
@@ -0,0 +1,84 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"sync"
+
+	"github.com/apache/rocketmq-client-go/v2/consumer"
+	"github.com/apache/rocketmq-client-go/v2/primitive"
+)
+
+func main() {
+	groupName := "testGroup"
+	c, err := consumer.NewManualPullConsumer(
+		consumer.WithGroupName(groupName),
+		consumer.WithNameServer(primitive.NamesrvAddr{"127.0.0.1:9876"}),
+	)
+	if err != nil {
+		log.Fatalf("init producer error: %v", err)
+	}
+
+	topic := "test"
+	// get all message queue
+	mqs, err := c.GetMessageQueues(context.Background(), topic)
+	if err != nil {
+		log.Fatalf("Get message queue error: %v", err)
+	}
+	var wg sync.WaitGroup
+
+	fn := func(mq *primitive.MessageQueue) {
+		defer wg.Done()
+		// get latest offset
+		offset, err := c.CommittedOffset(context.Background(), groupName, mq)
+		for {
+			if err != nil {
+				log.Fatalf("search latest offset error: %v", err)
+			}
+			// pull message
+			ret, err := c.PullFromQueue(context.Background(), groupName, mq, offset, 1)
+			if err != nil {
+				log.Fatalf("pullFromQueue error: %v", err)
+			}
+			if ret.Status == primitive.PullFound {
+				msgs := ret.GetMessageExts()
+				for _, msg := range msgs {
+					log.Printf("subscribe Msg: %v \n", msg)
+					// commit offset
+					if err = c.Seek(context.Background(), groupName, mq, msg.QueueOffset+1); err != nil {
```

Comment:
Using `Seek` to commit offset seems unclear because the `Seek` method is commonly used express reset a read/write position semantics. Could we rename this method to `CommitOffset` and rename `CommittedOffset ` to `QueryCommitedOffset` for reduce ambiguous? This is not fully considered thinking, what's your think about it?

Reply:
Well, I don't know how we talked about last month, as @vongosling mentioned in https://github.com/apache/rocketmq-client-go/issues/737, we should compatible same semantics with Java

Reply:
Well, I just remember what we last month talked about. as @vongosling mentioned in #737, We should stay compatible with Java Interface semantics. So, before I get a confirmation, you can ignore the conversation and pay attention to other conversion

Reply:
confirmation before release